modules/tinymce/src/core/test/ts/browser/focus/KeyboardShortcutMenuFocusTest.ts

```diff
@@ -0,0 +1,46 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { McEditor, TinyContentActions, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import { RawEditorOptions } from 'tinymce/core/api/OptionTypes';
+
+describe('browser.tinymce.themes.silver.editor.toolbar.KeyboardShortcutMenuFocusTest', () => {
+
+  const pTestFocus = async (options: RawEditorOptions) => {
+    const editor = await McEditor.pFromSettings<Editor>({
+      statusbar: false,
+      ...options,
+      base_url: '/project/tinymce/js/tinymce'
+    });
+    // currently for me i need to actually focus on the browser window for the test to pass, not sure if there's a bug somewhere
+    editor.focus();
```

Comment:
Looks like this was missed in the previous cleanup.

Reply:
Yeah, I tried to clean this up in one of the commits, and it resulted in all browser fail for this test

Reply:
it has been moved to editor config https://github.com/tinymce/tinymce/pull/7533/commits/7c91f0e22441844e9236c7393bd8e514e39968a0